### PR TITLE
fixes #4800 SLES special handling in facts parser

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: app/services/facts_parser.rb [hunk 1]
```diff
@@ -45,8 +45,12 @@ def operatingsystem
         os = Operatingsystem.find_by_name(os_name) || Operatingsystem.create!(:name => os_name)
       end
       if facts[:lsbdistdescription] && os.description.blank?
-        family = os.deduce_family || 'Operatingsystem'
-        os.description = family.constantize.shorten_description facts[:lsbdistdescription]
+        if os_name == 'SLES'
```
Comment (domcleal) [inline app/services/facts_parser.rb:48]: I think this should be outside the `if facts[:lsbdistdescription]` conditional, but inside one for `os.description.blank?` - maybe refactor it slightly?

Comment (mmoll) [inline app/services/facts_parser.rb:48]: refactored it slightly ;)

**Comment by dmitri-d** [comment]
lgtm.

**Comment by domcleal** [comment]
Merged as 07a1bb3 for Foreman 1.5.0, thanks @mmoll!

**Pull request closed without merging**